### ISSUE TITLE
[Bug]: Speaking papers full text not just selected text?

### ISSUE THREAD

**Issue by FloWstream-crypto** [ImperialSquid/zotero-zotts#88]
### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/ImperialSquid/zotero-zotts/issues) issues to see if someone else has already submitted the same bug report.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/ImperialSquid/zotero-zotts/issues?q=is%3Aissue+is%3Aclosed) issues to see if the problem is already solved and just waiting to be released.
- [X] I have described the problem with as much detail as possible.
- [X] If the bug only to occurs with a certain paper, I will include its details.

### Zotero version

7

### ZoTTS version

1.2.0 [edited by dev: fixed from 1.1.2 submitted, v1.1.2 doesn't exist...?]

### OS

Windows

### OS (specific)

Windows 11

### Expected behaviour

[Edited by dev: originally no response, reformatted from title]

Hello, so first of all, thanks for your work, I could have never done that! 

What I recognised, when I tried it today, that the tts just works when I mark part of the text and the press play. 

I use zotero 7 on my Windows 11 engine. 

I don't know if that is a general bug, or ifI made some kind of mistake. 

Thanks for the help!

**Comment by ImperialSquid** [comment]
Hi,

Just an fyi if you've never filled in a bug report, the box you put filled was just for the _title_, there's separate boxes further down the form for describing expected behaviour/actual behaviour/steps to reproduce...

---

In any case, it seems like you're asking "why can ZoTTS only speak a section of text when it's highlighted?"

This is a comment I've gotten a lot lol. Long story short is that accessing the full text of a paper looked to be more difficult than just accessing the selected text, so for the initial release I went with just speaking the selected text with full paper speaking to come later.

Now that it's been some time I've looked further into it and adding full text speaking should be doable, I just need to find time to code, test and release it! 

Hope that answers your question?
